src/clj/rems/db/catalogue.clj

```diff
@@ -22,7 +22,10 @@
   "Associates localisations into a catalogue item from
   the preloaded state."
   [item]
-  (assoc item :localizations ((cached :localizations) (:id item))))
+  (-> item
+      (assoc :localizations ((cached :localizations) (:id item))
+             :end (:endt item))
```

Comment:
We could rename the SQL column to `end` so we don't need this?

Reply:
Should endt be renamed to end in all tables? It’s in quite many places.

Reply:
End is a reserved word 
https://www.postgresql.org/docs/8.1/sql-keywords-appendix.html

Reply:
Yes it is but there is no need to avoid it . You can write `"end"`

Reply:
The rename of endt can be postponed to the cleaning phase next week if you add a task for it.

Reply:
I added a note to the ideas.